packages/orchestration/src/exos/chain-account-kit.js

```diff
@@ -188,15 +187,27 @@ export const prepareChainAccountKit = (zone, { watch, when }) =>
             chainId: this.state.chainId,
             addressEncoding: 'bech32',
           });
+          return Promise.resolve(watch(undefined));
         },
-        async onClose(_connection, reason) {
+        /**
+         * @param {Remote<Connection>} _connection
+         * @param {unknown} reason
+         * @returns {PromiseVow<void>}
+         */
+        onClose(_connection, reason) {
           trace(`ICA Channel closed. Reason: ${reason}`);
           // FIXME handle connection closing https://github.com/Agoric/agoric-sdk/issues/9192
           // XXX is there a scenario where a connection will unexpectedly close? _I think yes_
+          return Promise.resolve(watch(undefined));
         },
-        async onReceive(connection, bytes) {
+        /**
+         * @param {Remote<Connection>} connection
+         * @param bytes
+         * @returns {PromiseVow<string>}
+         */
+        onReceive(connection, bytes) {
           trace(`ICA Channel onReceive`, connection, bytes);
-          return '';
+          return Promise.resolve(watch(''));
```

Comment:
Should we instead do this?
```suggestion
          return watch(Promise.resolve(''));
```
Also, I'm not sure we need to return anything here - `return ''` might be an artifact from satisfying this type guard:
https://github.com/Agoric/agoric-sdk/blob/3027adf8613154dec167c5fccf5f207f6d2af701/packages/network/src/network.js#L1224

Reply:
There is no need for the `onReceive` handler altogether. I removed it in [d6bd228](https://github.com/Agoric/agoric-sdk/pull/9562/commits/d6bd228fac7d19b8924c92de5c64103ed15a25ef)